app/helpers/restaurant_helper.rb

```diff
@@ -0,0 +1,5 @@
+module RestaurantHelper
+  def create_dropdown_options
+    @select_options = Restaurant::PERMITTED_CATEGORIES.dup.insert(0, 'All Restaurants')
+  end
+end
```

Comment:
Mind the naming conventions... `RestaurantHelper` vs `RestaurantsHelper`
..and now we need a `restaurants_helper_spec.rb`